vpr/src/place/place.cpp

```diff
@@ -732,18 +890,37 @@ void try_place(const t_placer_opts& placer_opts,
 
         /* Run inner loop again with temperature = 0 so as to accept only swaps
          * which reduce the cost of the placement */
-        placement_inner_loop(&state, placer_opts,
-                             quench_recompute_limit, &stats,
-                             &costs,
-                             &moves_since_cost_recompute,
-                             pin_timing_invalidator.get(),
-                             place_delay_model.get(),
-                             placer_criticalities.get(),
-                             placer_setup_slacks.get(),
-                             *move_generator,
-                             blocks_affected,
-                             timing_info.get(),
-                             placer_opts.place_quench_algorithm);
+        if (placer_opts.place_quench_algorithm.is_timing_driven() && placer_opts.place_agent_multistate == true) {
+            placement_inner_loop(&state, placer_opts,
+                                 quench_recompute_limit, &stats,
+                                 &costs,
+                                 &moves_since_cost_recompute,
+                                 pin_timing_invalidator.get(),
+                                 place_delay_model.get(),
+                                 placer_criticalities.get(),
+                                 placer_setup_slacks.get(),
+                                 *move_generator2,
```

Comment:
Same as before; lots of duplicate code just to change one variable. This can be more succinct.

Reply:
Done